runtime/doc/options.txt

```diff
@@ -6014,7 +6013,6 @@ A jump table for the options with a short description can be found at |Q_op|.
 						*'statuscolumn'* *'stc'*
 'statuscolumn' 'stc'	string	(default "")
 			local to window
-	EXPERIMENTAL
```

Comment:
'statuscolumn' as been really stable AFAIK (thanks @luukvbaal !), and we don't plan to change its interface. Can probably graduate it?

Reply:
I tend to agree, however @bfredl wasn't happy with the fact that the statuscolumn width is estimated, and that a wrong estimation incurs a "panic" redraw. I think he went so far as to say that we couldn't graduate the feature without solving that but I don't really see a way around it (without sacrificing the "grow as needed" property it holds now).

The interface _might_ change the same way 'statusline' might, depending on the ongoing option work ("[Add a dict-based configuration format for complicated options such as winbar, tabline, statusline, etc](https://github.com/neovim/neovim/issues/25672)"). IDK if either of those reasons should actually prevent the feature from being marked stable.

Reply:
My concern remains, but I am not against removing the `EXPERIMENTAL` label as such. We don't have a "contract" for options the way we do for API functions, and it is reasonable to expect the interface to _evolve_ (with possibly lightly breaking changes, same as any other option) rather than being completely redesigned.